data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key/rotation/IndividualKeyRotationRequest.avsc

```diff
@@ -16,18 +16,23 @@
     },
     {
       "name": "oldParentKeyAlias",
-      "type": "string",
-      "doc": "The key alias whose protected content will be re-wrapped with a new key."
+      "type": ["null", "string"],
+      "doc": "For unmanaged key rotation. The key alias whose protected content will be re-wrapped with a new key."
```

Comment:
null means rewrap all matching wrapping keys?

Reply:
Yes, and for managed key rotation that is always the case. For unmanaged, not setting this would generate an error.